### PR TITLE
Upgrade ArgoCD to 1.8.1

### PR DESCRIPTION
**Pull request by anishasthana**
This PR will upgrade our ArgoCD to use the updated manifests from https://github.com/operate-first/continuous-deployment/releases/tag/v1.8.1-1

Signed-off-by: Anish Asthana <anishasthana1@gmail.com>

### CODE REVIEW

File: manifests/overlays/prod/statefulsets/argocd-application-controller_patch.yaml [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
-kind: Deployment
+kind: StatefulSet
 metadata:
   name: argocd-application-controller
 spec:
   template:
     spec:
       containers:
       - name: argocd-application-controller
-        image: quay.io/aicoe/argocd:v1.7.4-3
```
Comment (maulikjs) [inline manifests/overlays/prod/statefulsets/argocd-application-controller_patch.yaml:11]: dont we have to mention a new image here?

Comment (HumairAK) [reply manifests/overlays/prod/statefulsets/argocd-application-controller_patch.yaml:11]: yeah, these are now added from the base which is pulled remotely from here: 
https://github.com/operate-first/continuous-deployment/blob/master/manifests/base/kustomization.yaml#L13

Comment (anishasthana) [reply manifests/overlays/prod/statefulsets/argocd-application-controller_patch.yaml:11]: The image is actually specified in upstream manifests now.

**Review by HumairAK — APPROVED** [review]

File: manifests/overlays/prod/statefulsets/argocd-application-controller_patch.yaml (hunk 1 above)
Comment (maulikjs) [reply manifests/overlays/prod/statefulsets/argocd-application-controller_patch.yaml:11]: ah gotchu

**Review by maulikjs — APPROVED** [review]

**Pull request merged**